modP2pImpl/src/org/aion/p2p/impl1/tasks/TaskInbound.java

```diff
@@ -131,8 +131,8 @@ public void run() {
                             cb = (ChannelBuffer) key.attachment();
                             if (cb == null) {
                                 p2pLOG.error(
-                                        "inbound exception={}",
-                                        new P2pException("attachment is null").getMessage());
+                                        "inbound exception.",
+                                        new P2pException("attachment is null"));
```

Comment:
I don't think this exception is useful since it's only printed not thrown. We could just print the message without creating the exception object.